.circleci/config.yml

```diff
@@ -24,7 +24,12 @@ jobs:
           # fallback to using the latest cache if no exact match is found
           - v1-dependencies-
 
+      - run:
+          name: Install bower
+          command: 'sudo npm install -g bower'
+
       - run: yarn install
+      - run: bower install
```

Comment:
Is there some reason we should be using `bower`? It is still maintained, but they recommend [migrating away from it](https://bower.io/blog/2017/how-to-migrate-away-from-bower/). Can we install Slick some other way?